### ISSUE TITLE
[QUESTION]: <How can i connect YTMD Connector>

### ISSUE THREAD

**Issue by 1akaru1** [XeroxDev/YTMD-StreamDeck#145]
### Consent

- [X] I have searched the existing issues and verified that there is no open issue for the same subject.
- [X] I understand that the YTMDesktop Elgato Plugin has NO affiliation with Google, YouTube or the YTMDesktop application.

### Question

I've now tried to connect on my streamdeck, but I don't have a connect or authenticate button anywhere on streamdeck.
What should I do?
![111](https://github.com/user-attachments/assets/5f61d287-bb73-4598-9161-916b7de7f197)

**Comment by 1akaru1** [comment]
The YouTube app has allowed authentication, but it is not resolved.
![2222](https://github.com/user-attachments/assets/99ff1e0f-d4bb-4e76-b795-a613103811f1)

**Comment by 1akaru1** [comment]
I downloaded the plug-in, but how do I update it to the latest version?
I don't know how because I installed it from the plug-in collection on stream deck.

**Comment by XeroxDev** [comment]
If you've downloaded it through the Elgato Marketplace, here's a quick guide:

1. Click on the gear symbol
2. Click on the Plugins Tab and select the plugin
3. Click on the "Check for updates" button
![image](https://github.com/user-attachments/assets/0def8598-448b-4f33-99f8-59428287d982)

**Comment by 1akaru1** [comment]
i I'm using Monster Deck.
using monster deck.

**Comment by XeroxDev** [comment]
Sadly I'm not sure what a "Monster Deck" is, could you elaborate? Because in the description above, you've written "my streamdeck".

**Comment by 1akaru1** [comment]
https://www.monstargear.co.kr/1136/?idx=1010
https://www.monstargear.co.kr/505/?q=YToxOntzOjEyOiJrZXl3b3JkX3R5cGUiO3M6MzoiYWxsIjt9&bmode=view&idx=17314715&t=board

I'm using a stream deck from a company called Monsta Gear.
Attached is the url and software url of the company.

**Comment by XeroxDev** [comment]
I've investigated a bit and have to inform you that they don't seem to be affiliated or partnered with elgato. They also seem kinda shady/sketchy (disabled context menu, disabled the copy function, same Company Registration Number as other sketchy "companies", etc).

I've never developed a plugin for them and it seems they've just stolen mine and put it into their "collection" or something. Please correct me, if I'm wrong, but the fact you can't even update the plugin to the latest version and you're still on a such outdated one AND they advise you to run the software in administrator mode as soon as basic functions don't work (and many other things), also tells me a lot.

Therefore I've to inform you that I can't provide you any support for such devices, especially because I never developed anything for them, don't know their software and, this is the most important one, I don't trust them, nor do I support any kind copyright infringements.

# TL;DR
I've to inform you that I can't give you any support, because you're using an unsupported and unknown (to me) device manufacturer and software.

**Comment by 1akaru1** [comment]
Thank you for your response.
I'll have to contact the company about that.
Thank you for your kind and sincere response.
I hope you have a great day.
